unitTest/UtestEpubPacker.js

```diff
@@ -200,7 +200,7 @@ test("makeCoverImageXhtmlFile", function (assert) {
             "<body>" +
                "<div class=\"svg_outer svg_inner\">" +
                     "<svg xmlns=\"http://www.w3.org/2000/svg\" xmlns:xlink=\"http://www.w3.org/1999/xlink\" height=\"100%\" width=\"100%\" version=\"1.1\" preserveAspectRatio=\"xMidYMid meet\" viewBox=\"0 0 400 200\">" +
-                        "<image xlink:href=\"cover.png\" height=\"200\" width=\"400\"/>" +
+                        "<image xlink:href=\"...png\" height=\"200\" width=\"400\" data-origin=\"http://dummy/cover.png\"/>" +
```

Comment:
This only happens because your function throws an error saying its not implemented yet.